File: campeonato-front/src/app/models/dto/time-campeonato-dto.ts
export class TimeCampeonatoDTO {
  nomeTime: string;
  urlFotoTime: string;
  pontos: number;
  posicao: number;
  partidasJogadas: number;
  vitorias: number;
  empates: number;
  derrotas: number;
  golsMarcados: number;
  golsSofridos: number;
  saldoGols:number;


  constructor(nomeTime: string,
              urlFotoTime: string,
              pontos: number,
              posicao: number,
              partidasJogadas: number,
              vitorias: number,
              empates: number,
              derrotas: number,
              golsMarcados: number,
              golsSofridos: number,
              saldoGols: number) {
    this.nomeTime = nomeTime;
    this.urlFotoTime = urlFotoTime;
    this.pontos = pontos;
    this.posicao = posicao;
    this.partidasJogadas = partidasJogadas;
    this.vitorias = vitorias;
    this.empates = empates;
    this.derrotas = derrotas;
    this.golsMarcados = golsMarcados;
    this.golsSofridos = golsSofridos;
    this.saldoGols = saldoGols;
  }


}

File: campeonato-api/src/main/java/com/luizmarcelo/campeonatoapi/model/dto/TimeCampeonatoDTO.java
package com.luizmarcelo.campeonatoapi.model.dto;

import jakarta.persistence.Column;
import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class TimeCampeonatoDTO {
    private String nomeTime;
    private String urlFotoTime;
    private Integer pontos;
    private Integer posicao;
    private Integer partidasJogadas;
    private Integer vitorias;
    private Integer empates;
    private Integer derrotas;
    private Integer golsMarcados;
    private Integer golsSofridos;
    private Integer saldoGols;
    private Integer idTime;
}
